lib/src/rules/library_names.dart

```diff
@@ -7,12 +7,11 @@ import 'package:analyzer/dart/ast/visitor.dart';
 import 'package:linter/src/analyzer.dart';
 import 'package:linter/src/utils.dart';
 
-const _desc =
-    r'Name libraries and source files using `lowercase_with_underscores`.';
+const _desc = r'Name libraries using `lowercase_with_underscores`.';
 
 const _details = r'''
 
-**DO** name libraries and source files using `lowercase_with_underscores`.
+**DO** name libraries using `lowercase_with_underscores`.
 
 Some file systems are not case-sensitive, so many projects require filenames
 to be all lowercase.  Using a separate character allows names to still be
```

Comment:
I wonder if we should call out `file_names` as a related lint?

Reply:
done